test/e2e/basic_empty_dir_test.go

```diff
@@ -37,6 +37,15 @@ func TestBasicEmptyDir(t *testing.T) {
 			},
 			Replicas: 1,
 		},
+		Status: imageregistryapi.ImageRegistryStatus{
+			Storage: imageregistryapi.ImageRegistryConfigStorage{
+				Filesystem: &imageregistryapi.ImageRegistryConfigStorageFilesystem{
+					VolumeSource: corev1.VolumeSource{
+						EmptyDir: &corev1.EmptyDirVolumeSource{},
+					},
+				},
+			},
+		},
```

Comment:
why does this need to be passed in?

Reply:
I can remove that, i needed it before when i hard coded the NewDriverFromResource, will fix in all the tests

Reply:
updated in current push